data/dev.tchx84.Portfolio.desktop.in

```diff
@@ -4,7 +4,8 @@ Name=Portfolio
 Exec=dev.tchx84.Portfolio
 Terminal=false
 Type=Application
-Categories=Utility;
+GenericName=File Manager
+Categories=FileManager;
```

Comment:
No, this is also incorrect. Additional categories should always be used in combination with the main category, not alone. You should ideally use what I suggested in my previous comment: `Categories=GTK;Utility;FileManager;`

Reply:
@tchx84 said to use a single category in Categories.

Reply:
I said one main category :)

Reply:
@philipzae Could you please edit your last commit and change the Categories to `GTK;Utility;FileManager;`? Thanks!

Reply:
Done

Reply:
Thanks, LGTM now. :-)

Reply:
Thanks !!